cartoframes/contrib/vector.py

```diff
@@ -315,11 +365,17 @@ def __init__(self, dataframe, color=None, size=None, time=None,
 
 
 @utils.temp_ignore_warnings
-def vmap(layers,
-         context,
-         size=(1024, 632),
-         basemap=BaseMaps.voyager,
-         bounds=None):
+def vmap(
+        layers,
+        context,
+        carto_vl_path=_DEFAULT_CARTO_VL_PATH,
+        airship_path=None,
+        size=(1024, 632),
```

Comment:
I'm no expert on the UX/UI side of things, but I wonder if we should try to default to width=100% and let `size=None`. The user can override things if they want.

Reply:
I have to check because it depends on what Jupyter is wrapping this thing around in, but I think we might have to have a fixed size

Reply:
Checked it. We can, so I'm going to do the following:

- Will default to None, which implies having 100% width and 632px height.
- If the size is specified as a number, assume it's pixels
- If the size is specified as a string, assume it's valid CSS prop value. So you can do '50%' , or '10em', or whatever you heart desires